aws/rust-runtime/aws-config/src/profile/app_name.rs

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Load an app name from an AWS profile
+
+use crate::provider_config::ProviderConfig;
+use aws_sdk_sts::AppName;
```

Comment:
just to not confuse things, we should probably use the one from `aws-types`. We should probably `pub use` AppName from aws-config if we haven't already.

Reply:
Definitely. Auto-import kept choosing the re-export from STS, which was pretty frustrating.